lib/atlas/graph_persistor.rb

```diff
@@ -1,17 +1,20 @@
 module Atlas
   class GraphPersistor
-    def initialize(dataset, path)
-      @dataset = dataset
-      @path    = path
+    def initialize(dataset, path, export_modifier: nil)
```

Comment:
Nitpicking: I don't think `path` needs to be provided to `initialize`; it is only used in the `persist!` method. This is somewhat true of `export_modifier` too, but since that param forms part of the class "identity" I think it's fair to provide that when creating the class.

How about instead giving the destination path to persist?

```ruby
GraphPersitor.new(dataset, modifier).persist!(path)
```

---

At the moment, the whole class feels a bit overkill to me; it can be substituted for a three-line lambda:

```ruby
module Atlas
  GraphPersistor = lambda do |dataset, path, export_modifier: nil|
    data = EssentialExporter.dump(Runner.new(dataset).refinery_graph(:export))
    export_modifier.call(data) if export_modifier
    File.write(path, data.to_yaml)
  end
end
```

---

It could also use some documentation to explain what the params and return values are. `export_modifier` in particular is completely opaque to any new reader of this code. What values are yielded to the modifier? What should the modifier return?